recipes/nodejs/all/test_package/conanfile.py

```diff
@@ -1,12 +1,13 @@
-import os
-from conans import ConanFile, tools
+from conan import ConanFile
+from conan.tools.build import cross_building
 
 
 class TestPackageConan(ConanFile):
 
     settings = "os", "arch"
+    tool_requires = "nodejs/[*]@"
```

Comment:
This should be using `self_tested_reference` which you can find example of in the docs's template for pre compiled libraries

Reply:
You're right. It is removed now.

Reply:
Shall be resolved by: 6f4138b99ba39e6f7de52015104733b12186b012